chat-widget/src/components/headerstateful/HeaderStateful.tsx

```diff
@@ -36,10 +36,7 @@ export const HeaderStateful = (props: IHeaderStatefulParams) => {
             TelemetryHelper.logActionEvent(LogLevel.INFO, { Event: TelemetryEvent.HeaderCloseButtonClicked, Description: "Header Close button clicked." });
             if (conversationState.current === ConversationState.Active) {
                 dispatch({ type: LiveChatWidgetActionType.SET_SHOW_CONFIRMATION, payload: true });
-            } else if (conversationState.current === ConversationState.Postchat) {
```

Comment:
Why this change?

Reply:
This changed is to remove redundant logic now that I have moved the PostChat check into EndChat